tslint.json

```diff
@@ -11,7 +11,10 @@
   "rules": {
     "semicolon": false,
     "interface-name": false,
-    "no-console": false,
+    "no-console": {
+      "severity": "warning",
```

Comment:
Warnings have the tendency of being ignored and won't fail the lint. Do you think it's fine to just warn, or should you go with an error being thrown in your face?